src/components/Carousel/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, ReactChild } from 'react';
+import { StyleSheet, View } from 'react-native';
+import Button from 'components/Button';
+
+interface PropType {
+	children: ReactChild[];
+}
+
+export const Carousel: FC<PropType> = (props: PropType) => {

```

Comment:
- `props: PropType` -- `PropType` already introduced by `FC<PropType>`
 => just use `props` without re-clarify it's type.

- propose renam `PropType` to `Props` as we're using this naming convention in another places.